templates/default/tests/test_main.cairo

```diff
@@ -1,25 +1,28 @@
 %lang starknet
+from src.main import balance, increase_balance
+from starkware.cairo.common.cairo_builtins import HashBuiltin
 
-@contract_interface
-namespace BasicContract:
-    func increase_balance(amount : felt):
-    end
+@view
+func test_increasing_balance{syscall_ptr : felt*, range_check_ptr, pedersen_ptr : HashBuiltin*}():
+    let (result_before) = balance.read()
+    assert result_before = 0
 
-    func get_balance() -> (res : felt):
-    end
+    increase_balance(42)
+
+    let (result_after) = balance.read()
+    assert result_after = 42
+    return ()
 end
 
-@external
-func test_example{syscall_ptr : felt*, range_check_ptr}():
-    alloc_locals
+@view
+func test_failing_when_increasing_balance_with_negative_value{
```

Comment:
```suggestion
func test_revert_when_increasing_balance_with_negative_value{
```
failing may suggest test suppose to fail

Reply:
`test_revert` may suggest a test suppose to revert

I think I'll use `testCannot` (recommended by Foundry).

> A good practice is to use something like `testCannot` in combination with the [expectRevert](https://book.getfoundry.sh/cheatcodes/expect-revert.html) cheatcode